addon/components/frost-scroll.js

```diff
@@ -42,28 +53,28 @@ export default Component.extend(PropTypeMixin, {
   _setupPerfectScroll () {
     const debouncePeriod = 150
 
-    scheduleOnce('afterRender', this, () => {
+    run.scheduleOnce('afterRender', this, () => {
```

Comment:
b/c nested deconstructing is evil. I can add it back, but I would not add it as it was, I'd add it as two deconstructing lines.